src/engine/renderer/tr_image.cpp

```diff
@@ -1702,6 +1703,22 @@ static void R_LoadImage( const char **buffer, byte **pic, int *width, int *heigh
 		std::string altName = Str::Format("%s.%s", filename, imageLoaders[i].ext);
 		const FS::PakInfo* pak = FS::PakPath::LocateFile(altName);
 
+		// Darkplaces or Doom3 packages can ship alternative texture path in the form of
+		//   dds/<path without ext>.dds
+		if ( pak == nullptr && !Q_stricmp("dds", imageLoaders[i].ext) )
+		{
+			altName = Str::Format("dds/%s.%s", filename, imageLoaders[i].ext);
+			pak = FS::PakPath::LocateFile(altName);
+
+			// If this alternative path exists, it's expected to be loaded as the best one
```

Comment:
The `dds/` path will not be tried if there is a .dds in the regular path, which seems inconsistent with this comment. If the dds/ one should always be highest priority, I think it would be better to move the check before this loop.

Reply:
but doing it outside the loop makes sense :-)